OVERVIEW.md

```diff
@@ -236,7 +236,7 @@ Default: Infinity
 
 ### esnext
 
-Attempts to parse your code as ES6 using the harmony version of the esprima parser.
+Attempts to parse your code as ES6+ using the babel-jscs package as the parser.Please note that this is currently experimental, and will improve over time.
```

Comment:
`JSX` (uppercase?)
Not sure currently is needed.
"will improve" → "will be improved"?